instant-distance-py/Cargo.toml

```diff
@@ -11,12 +11,22 @@ repository = "https://github.com/InstantDomain/instant-distance"
 readme = "../README.md"
 
 [lib]
-name = "instant_distance"
-crate-type = ["cdylib"]
+name = "instant_distance_py"
```

Comment:
Renaming the crate (and related changes for packaging) should be in a different commit from the other changes.

Reply:
Should be in a separate commit now.